src/io/export_stl.cc

```diff
@@ -118,7 +118,7 @@ uint64_t append_stl(const PolySet& polyset, std::ostream& output, bool binary)
   if (!binary) {
     vertexStrings.resize(ps.vertices.size());
     std::transform(ps.vertices.begin(), ps.vertices.end(), vertexStrings.begin(),
-      [](const auto& p) { return toString({p.x(), p.y(), p.z()}); });
+      [](const auto& p) { return toString({(float) p.x(), (float) p.y(),(float)  p.z()}); });
```

Comment:
Instead of using C-style casts, I'd prefer using a C++ cast, e.g. `const auto f = p.cast<float>()`, then use the resulting float vector as arguments to `toString()`
```